src/main/scala/cromwell/binding/values/WdlFileLike.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.binding.values
+
+import cromwell.binding.types.WdlFileType
+
+/**
+ * WDL value which identifies the location of a File-like object.
+ */
+trait WdlFileLike extends WdlPrimitive {
+  val wdlType = WdlFileType
```

Comment:
But then if you're writing WDL you'd have to define output.txt as a FileLike to make it backend-agnostic? Unless we actually want to make it possible to explicitly specify File vs GCS output types in the WDL type system? (**Massive Caveat**: I might be totally misinterpreting the purpose of WdlTypes)